### ISSUE TITLE
Showcase section on the website for published papers using fortran code/packages

### ISSUE THREAD

**Issue by SiegristJ** [fortran-lang/fortran-lang.org#231]
Because of how central publishing is to the progress in science and academia, the website should include a section where it highlights new publications that build upon the good work of this fortran community. Whether for papers that specifically cite usage of packages from the package repository or papers that build upon fortran code, showcasing such work would be good both for authors and good for the fortran community.

**Comment by certik** [comment]
@SiegristJ thank you for opening up an issue for this. I personally think this is a great idea. We just have to setup criteria how to get a publication listed. One candidate could be https://github.com/fortran-lang/fortran-lang.org/pull/226.

**Comment by awvwgk** [comment]
For electronic structure theory this would probably lead to having the papers for [Vasp](https://vasp.at), [Turbomole](https://www.turbomole.org/), [CP2K](https://cp2k.org), [ADF](https://scm.com), [DFTB+](https://dftbplus.org), [Molpro](https://www.molpro.net/), [Molcas](http://www.molcas.org/), [BigDFT](http://bigdft.org), [ABINIT](https://www.abinit.org/), [Dalton](https://daltonprogram.org), [NWChem](https://nwchemgit.github.io), [QE](http://www.quantum-espresso.org/), [Siesta](https://departments.icmab.es/leem/siesta/), ... listed on the homepage and I'm certainly overlooking more than half of the relevant projects here for sure (I tried getting all open source ones in the package index already, but surely there are more).

**Comment by certik** [comment]
@awvwgk indeed, I don't know if it would show what we want to show and it would be a lot of papers, as Vasp alone is cited a lot.

**Comment by awvwgk** [comment]
Also, keep in mind that if we start to select a few software papers, we must have really good reasons to do so and prepare for the critique of the other active developer communities we might be “ignoring” with such a choice. Not a decision we can take lightly.

**Comment by certik** [comment]
We should also include all papers that use Python, because SciPy contains Fortran... I believe Julia and R also depend on Fortran, so we should include those papers also that use Julia or R.

Also, because Lapack's reference implementation is in Fortran, we should probably include all computational papers.

It's not clear to me where to draw the line.

**Comment by smeskos** [comment]
This is a great idea and feasible as long as we stick to papers that have verifiable citations to Fortran public and open-source codes and not those written primarily in other languages. Otherwise, and considering both @awvwgk's and @certik's valid concerns we should include, probably, half the computations papers ever published...  A good starting point would be to include only papers citing libraries from the fpm registry, which would also serve another purpose, to gradually push developers to make their libraries fpm-ready.

**Comment by SiegristJ** [comment]
One way that could make it easier to sort papers is if you put up on the website how to properly cite a fortran package in a publication. Then only include those papers that follow the guidelines. Would that not solve the problem of otherwise having to pick and choose who to include or to boil the ocean? @awvwgk @certik

**Comment by vmagnin** [comment]
> One way that could make it easier to sort papers is if you put up on the website how to properly cite a fortran package in a publication.

Note that packages on GitHub can obtain a DOI using a tool like Zenodo:
https://zenodo.org/record/2649855
If the software was not published in a paper, such a DOI it can be useful to cite the package.
